From: Hilko Bengen <bengen@debian.org>
Date: Thu, 23 May 2013 19:20:17 +0200
Subject: perl: Fix out-of-tree build by generating symlinks

---
 perl/Makefile.am | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/perl/Makefile.am b/perl/Makefile.am
index 49bf6cc..b49da00 100644
--- a/perl/Makefile.am
+++ b/perl/Makefile.am
@@ -57,9 +57,11 @@ all-local: Build
 
 clean-local: Build
 	./Build clean
+	if [ $(srcdir) != $(builddir) ]; then find -type l | xargs rm -f ; fi
 
 Build: Build.PL
-	$(PERL) $(srcdir)/Build.PL --prefix "@prefix@"
+	if [ $(srcdir) != $(builddir) ]; then cp -rsft $(builddir) $(abs_srcdir)/. ; fi
+	$(PERL) ./Build.PL --prefix "@prefix@"
 
 TESTS_ENVIRONMENT = $(top_builddir)/run --test
 
