From: =?UTF-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sat, 20 Feb 2010 19:06:21 +0100
Subject: [PATCH] scrub now available on Debian

---
 appliance/packagelist.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/appliance/packagelist.in b/appliance/packagelist.in
index 7d790f0..8d9522f 100644
--- a/appliance/packagelist.in
+++ b/appliance/packagelist.in
@@ -23,7 +23,6 @@
   nilfs-utils
   ntfsprogs
   reiserfs-utils
-  scrub
   libselinux
   udev
   util-linux-ng
@@ -62,6 +61,7 @@ ocfs2-tools
 */
 parted
 procps
+scrub
 strace
 xfsprogs
 zerofree
-- 
