From: Hilko Bengen <bengen@debian.org>
Date: Thu, 12 Dec 2013 21:06:22 +0100
Subject: golang: Fix for out-of-tree builds

---
 configure.ac       | 2 +-
 golang/Makefile.am | 2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 86cf440..fe92c16 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1442,7 +1442,7 @@ AS_IF([test "x$enable_golang" != "xno"],[
     AC_CHECK_PROG([GOLANG],[go],[go],[no])
     AS_IF([test "x$GOLANG" != "xno"],[
         AC_MSG_CHECKING([if $GOLANG is usable])
-        AS_IF([$GOLANG run golang/config-test.go 2>&AS_MESSAGE_LOG_FD],[
+        AS_IF([$GOLANG run $srcdir/golang/config-test.go 2>&AS_MESSAGE_LOG_FD],[
             AC_MSG_RESULT([yes])
 
             # Substitute some golang environment.
diff --git a/golang/Makefile.am b/golang/Makefile.am
index 58cafc6..ba13691 100644
--- a/golang/Makefile.am
+++ b/golang/Makefile.am
@@ -49,6 +49,7 @@ golangpkg_DATA = \
 	pkg/$(GOOS)_$(GOARCH)/$(pkg).a
 
 pkg/$(GOOS)_$(GOARCH)/$(pkg).a: src/$(pkg)/guestfs.go
+	-[ $(srcdir) != $(builddir) ] && ln -s $(abs_srcdir)/src $(builddir)/src
 	$(top_builddir)/run $(GOLANG) install $(pkg)
 
 golangsrc_DATA = $(source_files)
@@ -62,4 +63,5 @@ endif
 CLEANFILES = *~ src/$(pkg)/*~
 
 clean-local:
+	-[ $(srcdir) != $(builddir) ] && rm -f $(builddir)/src
 	rm -rf pkg
