From: Hilko Bengen <bengen@debian.org>
Date: Sun, 3 Jul 2011 12:47:23 +0200
Subject: Mount /run as tmpfs inside appliance (workaround for febootstrap's
 /init)

---
 appliance/init |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/appliance/init b/appliance/init
index f44ad7b..0c0ad76 100755
--- a/appliance/init
+++ b/appliance/init
@@ -16,6 +16,9 @@ rm -f /proc; mkdir /proc
 mount -t proc /proc /proc
 rm -f /sys; mkdir /sys
 mount -t sysfs /sys /sys
+# taken from initramfs-tools/init --Hilko Bengen
+mkdir -p /run
+mount -t tmpfs -o "nosuid,size=20%,mode=0755" tmpfs /run
 mkdir -p /run/lock
 
 # devtmpfs is required since udev 176
