From: Hilko Bengen <bengen@debian.org>
Date: Sun, 3 Jul 2011 12:47:23 +0200
Subject: Mount /run as tmpfs inside appliance (workaround for febootstrap's
 /init)

---
 appliance/init |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/appliance/init b/appliance/init
index 50dd76d..e60fa06 100755
--- a/appliance/init
+++ b/appliance/init
@@ -16,6 +16,10 @@ rm -f /proc; mkdir /proc
 mount -t proc /proc /proc
 rm -f /sys; mkdir /sys
 mount -t sysfs /sys /sys
+# taken from initramfs-tools/init --Hilko Bengen
+mkdir -p /run
+mount -t tmpfs -o "nosuid,size=20%,mode=0755" tmpfs /run
+mkdir -p /run/lock
 
 if [ ! -L /etc/init.d/udev -a -x /etc/init.d/udev ]; then
   if type service >/dev/null 2>&1; then
-- 
