From: Hilko Bengen <bengen@debian.org>
Date: Sun, 3 Jul 2011 12:47:23 +0200
Subject: Mount /run as tmpfs inside appliance (workaround for febootstrap's
 /init)

---
 appliance/init |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/appliance/init b/appliance/init
index 15ced71..c6853be 100755
--- a/appliance/init
+++ b/appliance/init
@@ -18,6 +18,9 @@ mount -t proc /proc /proc
 if [ ! -d /sys ]; then rm -f /sys; fi
 mkdir -p /sys
 mount -t sysfs /sys /sys
+# taken from initramfs-tools/init --Hilko Bengen
+mkdir -p /run
+mount -t tmpfs -o "nosuid,size=20%,mode=0755" tmpfs /run
 mkdir -p /run/lock
 
 # devtmpfs is required since udev 176
