From: Hilko Bengen <bengen@debian.org>
Date: Tue, 19 Jul 2011 20:48:13 +0200
Subject: configure.ac: do not generate debian/changelog

---
 configure.ac |    1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 09daf99..9e55043 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1129,7 +1129,6 @@ AC_CONFIG_FILES([Makefile
                  clone/Makefile
                  csharp/Makefile
                  daemon/Makefile
-                 debian/changelog
                  df/Makefile
                  edit/Makefile
                  erlang/Makefile
