From: =?UTF-8?q?Bastien=20ROUCARI=C3=88S?= <roucaries.bastien@gmail.com>
Date: Fri, 17 May 2013 10:11:37 +0200
Subject: remove last trace of ntfsprogs

---
 appliance/packagelist.in |    1 -
 1 file changed, 1 deletion(-)

diff --git a/appliance/packagelist.in b/appliance/packagelist.in
index 8a4a5ea..54895b8 100644
--- a/appliance/packagelist.in
+++ b/appliance/packagelist.in
@@ -79,7 +79,6 @@
   mtools
   nilfs-tools
   ntfs-3g
-  ntfsprogs
   openssh-client
   reiserfsprogs
   sysvinit /* for /sbin/reboot */
