From: Hilko Bengen <bengen@debian.org>
Date: Thu, 11 Aug 2011 00:24:11 +0200
Subject: Fix for systemd-based Debian systems

---
 appliance/make.sh.in |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/appliance/make.sh.in b/appliance/make.sh.in
index 0335f6e..d541227 100755
--- a/appliance/make.sh.in
+++ b/appliance/make.sh.in
@@ -63,4 +63,7 @@ if grep -q /usr/share/augeas/lenses supermin.d/hostfiles-t; then
     echo "/usr/share/augeas/lenses/*.aug" >> supermin.d/hostfiles-t
 fi
 
+# Make sure that /init can be run on systemd-enabled Debian systems
+echo '/lib/lsb/*' >> supermin.d/hostfiles-t
+
 mv supermin.d/hostfiles-t supermin.d/hostfiles
-- 
