From: Hilko Bengen <bengen@debian.org>
Date: Sat, 1 Mar 2014 17:41:08 +0100
Subject: Add systemd libraries (for guestfsd) to packagelist

---
 appliance/packagelist.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/appliance/packagelist.in b/appliance/packagelist.in
index b93cd09..5e2a1aa 100644
--- a/appliance/packagelist.in
+++ b/appliance/packagelist.in
@@ -69,6 +69,8 @@ ifelse(DEBIAN,1,
   libcap2
   libhivex0
   libpcre3
+  libsystemd-id128-0
+  libsystemd-journal0
   libyajl2
   linux-image
   dnl syslinux 'suggests' mtools, but in reality it's a hard dependency:
