From: Hilko Bengen <bengen@debian.org>
Date: Wed, 22 May 2013 19:22:47 +0200
Subject: Disable test-utimens, test-futimens (gnulib)

---
 gnulib/tests/Makefile.am |   16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/gnulib/tests/Makefile.am b/gnulib/tests/Makefile.am
index 1528ae5..ef3b57b 100644
--- a/gnulib/tests/Makefile.am
+++ b/gnulib/tests/Makefile.am
@@ -477,10 +477,10 @@ EXTRA_DIST += test-ftruncate.c test-ftruncate.sh signature.h macros.h
 
 ## begin gnulib module futimens-tests
 
-TESTS += test-futimens
-check_PROGRAMS += test-futimens
-test_futimens_LDADD = $(LDADD) $(LIB_CLOCK_GETTIME) $(LIB_NANOSLEEP) @LIBINTL@
-EXTRA_DIST += nap.h test-futimens.h test-utimens-common.h test-futimens.c signature.h macros.h
+# TESTS += test-futimens
+# check_PROGRAMS += test-futimens
+# test_futimens_LDADD = $(LDADD) $(LIB_CLOCK_GETTIME) $(LIB_NANOSLEEP) @LIBINTL@
+# EXTRA_DIST += nap.h test-futimens.h test-utimens-common.h test-futimens.c signature.h macros.h
 
 ## end   gnulib module futimens-tests
 
@@ -1665,10 +1665,10 @@ EXTRA_DIST += utimecmp.h
 
 ## begin gnulib module utimens-tests
 
-TESTS += test-utimens
-check_PROGRAMS += test-utimens
-test_utimens_LDADD = $(LDADD) $(LIB_CLOCK_GETTIME) $(LIB_NANOSLEEP) @LIBINTL@
-EXTRA_DIST += nap.h test-futimens.h test-lutimens.h test-utimens.h test-utimens-common.h test-utimens.c macros.h
+# TESTS += test-utimens
+# check_PROGRAMS += test-utimens
+# test_utimens_LDADD = $(LDADD) $(LIB_CLOCK_GETTIME) $(LIB_NANOSLEEP) @LIBINTL@
+# EXTRA_DIST += nap.h test-futimens.h test-lutimens.h test-utimens.h test-utimens-common.h test-utimens.c macros.h
 
 ## end   gnulib module utimens-tests
 
