From: Hilko Bengen <bengen@debian.org>
Date: Wed, 6 Jul 2011 09:30:04 +0200
Subject: configure.ac: use Python mechanism to determine PYTHON_INSTALLDIR

---
 configure.ac |   19 +++----------------
 1 files changed, 3 insertions(+), 16 deletions(-)

diff --git a/configure.ac b/configure.ac
index 544b434..6fd12ab 100644
--- a/configure.ac
+++ b/configure.ac
@@ -562,23 +562,10 @@ AS_IF([test "x$enable_python" != "xno"],
 			AC_MSG_NOTICE([Python install dir $PYTHON_INSTALLDIR])],
 			[PYTHON_INSTALLDIR=check])
 
+            AC_MSG_WARN([XXX PYTHON_INSTALLDIR=$PYTHON_INSTALLDIR])
 	    if test "x$PYTHON_INSTALLDIR" = "xcheck"; then
-	        PYTHON_INSTALLDIR=
-                for d in \
-                    $PYTHON_PREFIX/lib64/python$PYTHON_VERSION/site-packages \
-                    $PYTHON_PREFIX/lib/python$PYTHON_VERSION/site-packages \
-                    /usr/lib64/python$PYTHON_VERSION/site-packages \
-                    /usr/lib/python$PYTHON_VERSION/site-packages \
-                    /usr/local/lib/python$PYTHON_VERSION/site-packages
-                do
-                    AC_MSG_CHECKING([for $d])
-                    if test -d "$d"; then
-                        AC_MSG_RESULT([found])
-                        PYTHON_INSTALLDIR=$d
-                        break
-                    fi
-                    AC_MSG_RESULT([not found])
-                done
+                PYTHON_INSTALLDIR=`$PYTHON -c "import distutils.sysconfig; \
+                print (distutils.sysconfig.get_python_lib(1,0));"`
                 if test "x$PYTHON_INSTALLDIR" = "x"; then
                     AC_MSG_WARN([Python site-packages directory not found])
                 fi
-- 
