From: Hilko Bengen <bengen@debian.org>
Date: Mon, 15 Aug 2011 00:48:11 +0200
Subject: out-of-tree build: daemon

---
 daemon/Makefile.am |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/daemon/Makefile.am b/daemon/Makefile.am
index 02c21df..c33758f 100644
--- a/daemon/Makefile.am
+++ b/daemon/Makefile.am
@@ -42,10 +42,10 @@ noinst_LIBRARIES = libprotocol.a
 libprotocol_a_SOURCES = guestfs_protocol.c guestfs_protocol.h
 libprotocol_a_CFLAGS = -Wall -Wno-unused -fno-strict-aliasing
 
-guestfs_protocol.c: $(libsrcdir)/guestfs_protocol.c
+guestfs_protocol.c: $(srcdir)/guestfs_protocol.c
 	rm -f $@
 	ln $< $@
-guestfs_protocol.h: $(libsrcdir)/guestfs_protocol.h
+guestfs_protocol.h: $(srcdir)/guestfs_protocol.h
 	rm -f $@
 	ln $< $@
 $(libsrcdir)/guestfs_protocol.c: force
