From: Hilko Bengen <bengen@debian.org>
Date: Thu, 17 Nov 2011 23:56:15 +0100
Subject: ruby: install to vendor_ruby.

---
 ruby/Makefile.am |   12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/ruby/Makefile.am b/ruby/Makefile.am
index c4b680f..db6ec9e 100644
--- a/ruby/Makefile.am
+++ b/ruby/Makefile.am
@@ -54,13 +54,13 @@ all: $(generator_built)
 	$(RAKE) build
 	$(RAKE) rdoc
 
-RUBY_SITELIB := $(shell $(RUBY) -rrbconfig -e "puts RbConfig::CONFIG['sitelibdir']")
-RUBY_SITEARCH := $(shell $(RUBY) -rrbconfig -e "puts RbConfig::CONFIG['sitearchdir']")
+RUBY_VENDORLIB := $(shell $(RUBY) -rrbconfig -e "puts RbConfig::CONFIG['vendorlibdir']")
+RUBY_VENDORARCH := $(shell $(RUBY) -rrbconfig -e "puts RbConfig::CONFIG['vendorarchdir']")
 
 install:
-	$(MKDIR_P) $(DESTDIR)$(RUBY_SITELIB)
-	$(MKDIR_P) $(DESTDIR)$(RUBY_SITEARCH)
-	$(INSTALL) -p -m 0644 $(srcdir)/lib/guestfs.rb $(DESTDIR)$(RUBY_SITELIB)
-	$(INSTALL) -p -m 0755 $(builddir)/ext/guestfs/_guestfs.so $(DESTDIR)$(RUBY_SITEARCH)
+	$(MKDIR_P) $(DESTDIR)$(RUBY_VENDORLIB)
+	$(MKDIR_P) $(DESTDIR)$(RUBY_VENDORARCH)
+	$(INSTALL) -p -m 0644 $(srcdir)/lib/guestfs.rb $(DESTDIR)$(RUBY_VENDORLIB)
+	$(INSTALL) -p -m 0755 $(builddir)/ext/guestfs/_guestfs.so $(DESTDIR)$(RUBY_VENDORARCH)
 
 endif
