From: Hilko Bengen <bengen@hilluzination.de>
Date: Sun, 2 Mar 2014 19:18:18 +0100
Subject: Fix out-of-tree appliance build

---
 appliance/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/appliance/Makefile.am b/appliance/Makefile.am
index ef8af5e..e38c534 100644
--- a/appliance/Makefile.am
+++ b/appliance/Makefile.am
@@ -106,7 +106,7 @@ daemon.tar.gz: ../daemon/guestfsd guestfsd.suppressions
 
 init.tar.gz: init
 	rm -f $@ $@-t
-	tar zcf $@-t init
+	( cd $(srcdir) && tar zcf - init ) > $@-t
 	mv $@-t $@
 
 # We should put this file in /lib/udev/rules.d, but put it in /etc so
