From: Hilko Bengen <bengen@debian.org>
Date: Fri, 21 Dec 2012 00:23:30 +0100
Subject: erlang: fix bindtests for out-of-tree build

---
 erlang/run-bindtests |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/erlang/run-bindtests b/erlang/run-bindtests
index 1a6aea5..17db79e 100755
--- a/erlang/run-bindtests
+++ b/erlang/run-bindtests
@@ -19,5 +19,5 @@
 set -e
 
 # The output is sent to bindtests.tmp.
-./bindtests.erl
+$srcdir/bindtests.erl
 diff -u $srcdir/../bindtests bindtests.tmp
