From: Hilko Bengen <bengen@debian.org>
Date: Mon, 9 Jul 2012 19:18:17 +0200
Subject: podwrapper.sh: Use date from ChangeLog

---
 podwrapper.sh.in |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/podwrapper.sh.in b/podwrapper.sh.in
index ec750f1..0a84d13 100755
--- a/podwrapper.sh.in
+++ b/podwrapper.sh.in
@@ -53,6 +53,9 @@ if [ -z "$abs_top_builddir" ]; then
     exit 1
 fi
 
+DATE=`awk '/^[0-9]+-[0-9]+-[0-9]+/ { print $1; exit }' \
+        $abs_top_srcdir/ChangeLog`
+
 declare -a inserts
 declare -a pattern
 declare -a indent
@@ -167,6 +170,7 @@ fi
 # Now generate the final output format(s).
 if [ -n "$man_output" ]; then
     "$POD2MAN" "$POD2_STDERR_OPTION" "$POD2_UTF8_OPTION" \
+        --date $DATE \
         --section "$section" -c "Virtualization Support" --name "$name" \
         --release "$PACKAGE_NAME-$PACKAGE_VERSION" \
         < $tmpdir/full.pod > "$man_output".tmp
