From: Hilko Bengen <bengen@debian.org>
Date: Sat, 18 Aug 2012 13:26:19 +0200
Subject: The package containing the diff binary has been diffutils for years

---
 appliance/packagelist.in |    4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/appliance/packagelist.in b/appliance/packagelist.in
index 15b3f41..120bd86 100644
--- a/appliance/packagelist.in
+++ b/appliance/packagelist.in
@@ -23,7 +23,6 @@
   btrfs-progs
   cryptsetup
   cryptsetup-luks /* old name used before Fedora 17 */
-  diffutils
   e2fsprogs
   /* e4fsprogs only exists on RHEL 5, will be ignored everywhere else. */
   e4fsprogs
@@ -50,7 +49,6 @@
   bsdmainutils
   btrfs-tools
   cryptsetup
-  diff
   e2fsprogs
   gfs-tools
   gfs2-tools
@@ -75,7 +73,6 @@
   vim
   btrfs-progs-unstable
   cryptsetup
-  diffutils
   augeas
   zfs-fuse
   e2fsprogs
@@ -96,6 +93,7 @@ binutils
 bzip2
 coreutils
 cpio
+diffutils
 dosfstools
 file
 findutils
