From: Hilko Bengen <bengen@debian.org>
Date: Fri, 21 Oct 2011 23:46:24 +0200
Subject: Fix --{en,dis}able-install-daemon

---
 configure.ac |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure.ac b/configure.ac
index 897ab32..5c2b9cc 100644
--- a/configure.ac
+++ b/configure.ac
@@ -250,9 +250,9 @@ if test "x$enable_daemon" = "xyes"; then
                   [enable installing the daemon under $sbindir @<:@default=no@:>@])],
                 [],
                 [enable_install_daemon=no])
-        AM_CONDITIONAL([INSTALL_DAEMON],[test "x$enable_install_daemon" = "xyes"])
         AC_MSG_RESULT([$enable_install_daemon])
 fi
+AM_CONDITIONAL([INSTALL_DAEMON],[test "x$enable_install_daemon" = "xyes"])
 
 dnl Build the appliance?
 AC_MSG_CHECKING([if we should build the appliance])
-- 
