From: Hilko Bengen <bengen@debian.org>
Date: Tue, 27 Mar 2012 19:36:54 +0200
Subject: Fix for building appliance out-of-tree

---
 appliance/Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/appliance/Makefile.am b/appliance/Makefile.am
index 5114275..6d256c4 100644
--- a/appliance/Makefile.am
+++ b/appliance/Makefile.am
@@ -71,7 +71,7 @@ supermin.d/daemon.img: ../daemon/guestfsd guestfsd.suppressions
 	rm -rf tmp
 	mkdir -p tmp$(DAEMON_SUPERMIN_DIR) tmp/etc
 	ln ../daemon/guestfsd tmp$(DAEMON_SUPERMIN_DIR)/guestfsd
-	ln guestfsd.suppressions tmp/etc/guestfsd.suppressions
+	ln @srcdir@/guestfsd.suppressions tmp/etc/guestfsd.suppressions
 	( cd tmp && find | cpio --quiet -o -H newc ) > $@-t
 	rm -rf tmp
 	mv $@-t $@
