From: Hilko Bengen <bengen@debian.org>
Date: Wed, 23 May 2012 22:13:11 +0200
Subject: Fix generation of gettext stubs for out-of-tree builds

---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 5a969c2..c11a600 100644
--- a/configure.ac
+++ b/configure.ac
@@ -815,7 +815,7 @@ AS_IF([test "x$OCAMLC" != "xno"],[
     dnl Write gettext modules for each OCaml tool.  If OCaml gettext
     dnl is not available then we write dummy functions.
     for program in resize sparsify sysprep; do
-        output=$program/${program}_gettext.ml
+        output=$builddir/$program/${program}_gettext.ml
         AC_MSG_NOTICE([creating $output])
         rm -f $output
 
