From: Hilko Bengen <bengen@debian.org>
Date: Fri, 5 Aug 2011 10:18:02 +0100
Subject: python: don't build static library, don't build .so.0.0, symlinks

---
 python/Makefile.am |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/python/Makefile.am b/python/Makefile.am
index 21db292..f6ad8da 100644
--- a/python/Makefile.am
+++ b/python/Makefile.am
@@ -43,6 +43,7 @@ libguestfsmod_la_SOURCES = guestfs-py.c guestfs-py.h guestfs-py-byhand.c
 libguestfsmod_la_CFLAGS = -Wall -I$(PYTHON_INCLUDEDIR) \
 			  -I$(top_srcdir)/src -I$(top_builddir)/src
 libguestfsmod_la_LIBADD = $(top_builddir)/src/libguestfs.la
+libguestfsmod_la_LDFLAGS = -avoid-version -shared
 
 TESTS_ENVIRONMENT = \
 	LIBGUESTFS_PATH=$(top_builddir)/appliance \
-- 
