From: Hilko Bengen <bengen@debian.org>
Date: Mon, 14 Aug 2017 21:39:58 +0200
Subject: Add fdisk to appliance list

---
 appliance/packagelist.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/appliance/packagelist.in b/appliance/packagelist.in
index e00eae2..8c188be 100644
--- a/appliance/packagelist.in
+++ b/appliance/packagelist.in
@@ -60,6 +60,7 @@ dnl old name used in Jessie and earlier
   btrfs-tools
   cryptsetup
   extlinux
+  fdisk
   genisoimage
 dnl gfs-tools, gfs2-tools have been renamed to gfs2-utils
   gfs-tools
