From: Hilko Bengen <bengen@hilluzination.de>
Date: Mon, 11 Jun 2012 00:09:34 +0200
Subject: 1:1.18.1-1

---
 debian/changelog |    9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/debian/changelog b/debian/changelog
index c91ba2f..02491b5 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,12 @@
+libguestfs (1:1.18.1-1) unstable; urgency=low
+
+  * New upstream version
+  * [9449d4e] Updated watch file
+  * [a8bb722] Added a method for determining available Ruby versions at build time
+  * [ff273f8] rebased patch queue
+
+ -- Hilko Bengen <bengen@debian.org>  Mon, 11 Jun 2012 00:08:57 +0200
+
 libguestfs (1:1.18.0-1) unstable; urgency=low
 
   * New upstream version
