From: Hilko Bengen <bengen@debian.org>
Date: Sun, 8 Jan 2017 14:18:18 +0100
Subject: Add ldmtool to appliance

---
 appliance/packagelist.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/appliance/packagelist.in b/appliance/packagelist.in
index e21c74047..23a259943 100644
--- a/appliance/packagelist.in
+++ b/appliance/packagelist.in
@@ -74,6 +74,7 @@ dnl iproute has been renamed to iproute2
   iputils-arping
   iputils-tracepath
   isc-dhcp-client
+  ldmtool
   libaugeas0
   libc-bin
   libcap2
