From: Hilko Bengen <bengen@debian.org>
Date: Wed, 19 Oct 2011 21:41:12 +0200
Subject: out-of-tree build: fix HAVE_OCAML=false case

---
 generator/Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/generator/Makefile.am b/generator/Makefile.am
index a127a87..b967a0b 100644
--- a/generator/Makefile.am
+++ b/generator/Makefile.am
@@ -85,7 +85,7 @@ else
 # No OCaml compiler.  Just replace the generator with a script that
 # touches stamp-generator and prints a warning.
 
-generator: $(SOURCES)
+$(srcdir)/generator: $(SOURCES)
 	rm -f $@ $@-t
 	echo 'echo Warning: Install OCaml compiler in order to rebuild the generated files.' >> $@-t
 	echo 'touch generator/stamp-generator' >> $@-t
-- 
