From: Hilko Bengen <bengen@hilluzination.de>
Date: Mon, 15 Aug 2011 00:48:11 +0200
Subject: out-of-tree build: daemon

---
 daemon/Makefile.am |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/daemon/Makefile.am b/daemon/Makefile.am
index 3fe8c88..c6c3cef 100644
--- a/daemon/Makefile.am
+++ b/daemon/Makefile.am
@@ -52,10 +52,10 @@ noinst_LIBRARIES = libprotocol.a
 libprotocol_a_SOURCES = guestfs_protocol.c guestfs_protocol.h
 libprotocol_a_CFLAGS = -Wall -Wno-unused -fno-strict-aliasing
 
-guestfs_protocol.c: $(libsrcdir)/guestfs_protocol.c
+guestfs_protocol.c: $(srcdir)/guestfs_protocol.c
 	rm -f $@
 	ln $< $@
-guestfs_protocol.h: $(libsrcdir)/guestfs_protocol.h
+guestfs_protocol.h: $(srcdir)/guestfs_protocol.h
 	rm -f $@
 	ln $< $@
 $(libsrcdir)/guestfs_protocol.c: force
-- 
