From: Hilko Bengen <bengen@debian.org>
Date: Thu, 7 Mar 2013 22:00:16 +0100
Subject: out-of-tree build: Fix link creation

---
 appliance/Makefile.am |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/appliance/Makefile.am b/appliance/Makefile.am
index 8661bf4..7a30b0e 100644
--- a/appliance/Makefile.am
+++ b/appliance/Makefile.am
@@ -108,7 +108,7 @@ supermin.d/udev-rules.img: 99-guestfs-serial.rules
 	rm -f $@ $@-t
 	rm -rf tmp-u
 	mkdir -p tmp-u/etc/udev/rules.d
-	for f in $^; do ln $$f tmp-u/etc/udev/rules.d/$$f; done
+	for f in $^; do ln $$f tmp-u/etc/udev/rules.d/$$(basename $$f); done
 	( cd tmp-u && find | cpio --quiet -o -H newc ) > $@-t
 	rm -rf tmp-u
 	mv $@-t $@
