From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Wed, 2 Nov 2011 18:19:15 +0000
Subject: python: Include <config.h>.

Ooops ...
---
 generator/generator_python.ml |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/generator/generator_python.ml b/generator/generator_python.ml
index c8448aa..10a4eca 100644
--- a/generator/generator_python.ml
+++ b/generator/generator_python.ml
@@ -35,6 +35,8 @@ let rec generate_python_c () =
   generate_header CStyle LGPLv2plus;
 
   pr "\
+#include <config.h>
+
 #include <stdio.h>
 #include <stdlib.h>
 #include <assert.h>
-- 
