From: Hilko Bengen <bengen@debian.org>
Date: Sat, 12 Nov 2011 21:11:26 +0100
Subject: Install Java bindings to jni subdirectory

---
 java/Makefile.am |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/java/Makefile.am b/java/Makefile.am
index ef78773..b76e65c 100644
--- a/java/Makefile.am
+++ b/java/Makefile.am
@@ -17,6 +17,8 @@
 
 include $(top_srcdir)/subdir-rules.mk
 
+libdir = ${prefix}/lib/jni
+
 # Old RHEL 5 autoconf doesn't have builddir.
 builddir ?= $(top_builddir)/java
 
-- 
