--- a/ocaml-dep.sh.in
+++ b/ocaml-dep.sh.in
@@ -49,10 +49,10 @@ ocaml
 # Output file is always created in the current directory.
 output=.depend
 
-subdir=$(realpath --relative-to=@abs_top_builddir@ .)
-srcdir=$(realpath --relative-to=. @abs_top_srcdir@/${subdir})
-srcdir_re=$(realpath --relative-to=. @abs_top_srcdir@/${subdir} | sed 's/\./[.]/g')
-top_builddir=$(realpath --relative-to=. @abs_top_builddir@)
+subdir=$(python -c "import os.path; print os.path.relpath('.', '@abs_top_builddir@')")
+srcdir=$(python -c "import os.path; print os.path.relpath('@abs_top_srcdir@/${subdir}', '.')")
+srcdir_re=$(echo ${srcdir} | sed 's/\./[.]/g')
+top_builddir=$(python -c "import os.path; print os.path.relpath('@abs_top_builddir@', '.')")
 
 includes="-I @abs_top_srcdir@/$subdir -I @abs_top_builddir@/$subdir"
 for i in $include_dirs; do
